opensync-ext-static/src/main/resources/ProfileRf.json

```diff
@@ -0,0 +1,86 @@
+{
+	"model_type": "Profile",
+	"id": 9,
+	"customerId": 2,
+	"profileType": "rf",
+	"name": "DefaultRf-2g",
```

Comment:
To what does "2g" refer in this name? Radio type?

Reply:
That was probably added when I ran my local tests and picked this file up as an example of a complete RF profile json. It isn't critical, but it should be clarified to reflect a default RF profile that contains a config for each valid radioType instead of just 2G.

Reply:
These are just the names of the static profiles that are read in for the static os gateway.
This profile, being for all the radios in the list, should probably just change the name to DefaultRf?
However, the name has no affect on the functionality. And this is only for testing (the static gateway)